components/ontology-access/api/src/main/java/org/phenotips/ontology/internal/solr/SolrOntologyTerm.java

```diff
@@ -220,6 +241,31 @@ private long processAncestorsAtDistance(int localDistance, Set<OntologyTerm> sou
     }
 
     @Override
+    public JSON toJson() throws Exception
+    {
+        JSONObject json = new JSONObject();
+
+        Iterator<Map.Entry<String, Object>> docIterator = this.doc.iterator();
+
+        while (docIterator.hasNext()) {
+            Map.Entry<String, Object> param = docIterator.next();
```

Comment:
The convention is that `thingIterator` describes what are the values returned by the iterator, not the type that contains the things to be iterated. So `docIterator` sounds like it iterates over a list of documents. It should be named `fieldIterator`.